routes/api.js

```diff
@@ -19,15 +19,9 @@ router.get('/', function(req, res, next) {
 router.use(isLoggedIn);
 
 function isLoggedIn(req, res, next) {
-	if (req.isAuthenticated()) {
-		return next();
-	}
-	// if they are not logged in, save the
-	// current page they're trying to get to
-	// this way they can come back after authenticating
-	req.session.returnTo = "/api"+req.url;
-	console.log("User is not logged in.");
-	res.redirect('/login');
+	if (!req.isAuthenticated()) {
+		res.status(401);
```

Comment:
this should be return res.status(401);, otherwise it will return a non-authenticated status but then still allow the user to view the resource.